pkgs/by-name/cr/crowdsec-firewall-bouncer/package.nix

```diff
@@ -0,0 +1,54 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  versionCheckHook,
+  nix-update-script,
+}:
+buildGoModule (finalAttrs: {
+  pname = "crowdsec-firewall-bouncer";
+  version = "0.0.33";
+
+  src = fetchFromGitHub {
+    owner = "crowdsecurity";
+    repo = "cs-firewall-bouncer";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-4fxxAW2sXGNxjsc75fd499ciuN8tjGqlpRIaHYUXwQ0=";
+  };
+
+  vendorHash = "sha256-Bhp6Z2UlCJ32vdc3uINCGleZFP2WeUn/XK+Q29szUzQ=";
+
+  ldflags = [
+    "-X github.com/crowdsecurity/go-cs-lib/version.Version=v${finalAttrs.version}"
```

Comment:
I went to check upstream and they do have the v prefix too :+1: 

```
./crowdsec-firewall-bouncer --version
version: v0.0.33-cb8b3e3c654499f745ff487eb1c327d7234a533f
BuildDate: 2025-05-20_08:01:34
GoVersion: 1.24.2
Platform: linux
```

If we want to add the commit to the end and at least set build date to the date on the latest commit we can extract that info but it's easier to leave it as-is for now.
https://github.com/NixOS/nixpkgs/blob/3e3afe5174c561dee0df6f2c2b2236990146329f/pkgs/by-name/ve/vexctl/package.nix#L18-L27